code/pagetypes/Forum.php

```diff
@@ -107,16 +107,16 @@ function canModerate($member = null) {
 		if(!$member) $member = Member::currentUser();
 
 		if(!$member) return false;
-		
+
 		// Admins
-		if ($this->canEdit($member)) return true; 
+		if (Permission::checkMember($member, 'ADMIN')) return true;
```

Comment:
We could, maybe that can be a new feature... for now keen to just get 0.8 PRs cleared and released.